server/src/main/java/org/opensearch/index/analysis/Analysis.java

```diff
@@ -245,17 +245,15 @@ public static <T> List<T> parseWordList(
         int lineNum = 0;
         for (String word : words) {
             lineNum++;
-            if (word.startsWith("#") == false) {
-                try {
-                    rules.add(parser.apply(word));
-                } catch (RuntimeException ex) {
-                    String wordListPath = settings.get(settingPath, null);
-                    if (wordListPath == null || isUnderConfig(env, wordListPath)) {
-                        throw new RuntimeException("Line [" + lineNum + "]: " + ex.getMessage());
-                    } else {
-                        LOGGER.error("Line [{}]: {}", lineNum, ex);
-                        throw new RuntimeException("Line [" + lineNum + "]: " + "Invalid rule");
-                    }
+            try {
```

Comment:
@dblock I think this is quite an impact change for existing users  - instead of ignoring the comment we would suddenly treat every comment as hashtag?

Reply:
Yes yes, but this was introduced accidentally in 2.x I believe and was a new undocumented feature, breaking support for hash tags. That's why I asked for the timeline of the above.

Reply:
:+1: